cpp/ql/src/semmle/code/cpp/Function.qll

```diff
@@ -20,6 +20,7 @@ class Function extends Declaration, ControlFlowNode, AccessHolder, @function {
   override string getName() { functions(underlyingElement(this),result,_) }
 
   /**
+   * DEPRECATED: Use `getIdentityString(Declaration)` from `semmle.code.cpp.Print` instead.
```

Comment:
FWIW, this will be less discoverable via autocomplete.

Reply:
It's my intention that the predicate should be hard to discover if it's infeasible to compute on large snapshots. I don't know how feasible it was to compute `getIdentityString`, though.